vectors/s206.fct

```diff
@@ -0,0 +1,78 @@
+# Test for S206, Dual Flip-Flop
+
+# inputs
+# flip flop 1
+# L and K is inputs
+# D is clock pulse input
+# outputs are J and E
+
+# flip flop 2
+# U and T is inputs
+# N is clock pulse input
+# Outputs are P and R
+# General clear input on pin F
+
+
+# -3V = 1
+# 0V = 0
+
+# Define the limits when testing inputs
+input-current='-1.0mA'
+input-current-margin='20.0mA'
+input-active-level='high'
+input-logic-high='-300mV'
```

Comment:
I thing this value are a bit to low. Use -50mV.
The value is being substrated and the differens should not be more that +/- 100mV. So 300mV will fail the input tests.